user-impl/src/test/java/com/example/auction/user/impl/UserRepositoryTest.java

```diff
@@ -56,8 +56,12 @@ public static void afterAll() {
     private final String email = "admin@gmail.com";
     private final String password = PUserCommand.hashPassword("admin");
     private final Instant createdAt = Instant.now();
-
+    int size = 25;
+    PaginatedSequence<User> createdUsers = Await.result(userRepository.getUsers(1, 10));
     private final PUser userCreated = new PUser(userId, createdAt, name, email, password);
+    int initialCount = createdUsers.getCount();
```

Comment:
Looking back to the code I just pasted here, it is even better to call `userRepository.countUsers()` instead of `userRepository.getUsers(1, 10)`.  Certainly for the first call where we are only interested in the total current count.